communitiesuk/funding-service-design-frontend
# Fs 2514 multi fund dashboard

https://digital.dclg.gov.uk/jira/browse/FS-2514


### Change description
Updating the dashboard page (frontend/account) to provide a 'view all applications' option, as per this mural https://app.mural.co/t/mhclg2837/m/mhclg2837/1675177182644/578e9019ed4a6da033b5de2183289ca7298e5f12?sender=u94fce4b3d87aef5380039386.

Dashboard has been split into 2 html pages - `dashboard_single_fund.html` is the current dashbaord, `dashboard_all.html` is for all applications.



### How to test
test the dashboard as normal, a view all applications link should display on the header next to the signout link. If you want to see this feature toggled off (as it would be in production for now) change the feature config value in default config to use the prod feature config from common config (can check in production config to see how this is done)


### Screenshots of UI changes (if applicable)
